devicon.json

```
@@ -1594,6 +1594,39 @@
         "color": "#f47c00",
         "aliases": []
     },
+    {
+        "name": "corejs",
+        "altnames": [
+            "core-js",
+            "core.js"
+        ],
+        "tags": [
+            "javascript",
+            "utility",
+            "polyfill"
+        ],
```

There are some new tags here: `polyfill` and `utility`
Should we add those new tags @Snailedlt?
```suggestion
        "tags": [
            "javascript",
            "library",
            "polyfill",
            "tool",
            "utility"
        ],
```

---

hmm, I don't think `utility` should be added, since most of the icons here can be considered utility tools, so we would have to add it pretty much everywhere.

[`polyfill`](https://developer.mozilla.org/en-US/docs/Glossary/Polyfill) seems like a sensible tag to add though.

---

So, all the tags except `utility`? or should I remove `tool` as well?

---

Yeah, all except `utility` :)
`tool` can be kept, since it's already used in `devicon.json` :)

---

Suggestion updated! 😀